### ISSUE TITLE
[Keyboard Navigation - MyAccess - Filters] Filter controls are not accessible through keybaord on filters screen.

### ISSUE THREAD

**Issue by sonimishra** [OfficeDev/office-ui-fabric-react#10415]
**User Experience:**
User rely on keyboard will find difficulty to interact with the control if that control is not accessible through keyboard and also won't be able to perform functionality over such control.

**MAS Reference:** 
MAS 2.1.1 - Keyboard (36)

**Repro steps:**
1. Open URL 'https://myaccess.microsoft.com/@M365x202152.onmicrosoft.com#/access-packages' and enter valid credentials.
2. Navigate to 'Request History' list item and select it.​
3. 'Request History' screen will open.​
4. Navigate to ' Filter' button and select it.
5. 'Filters' screen will open.​
6. Navigate to various controls like Submitted, Pending approval etc. under status lying on filters screen.
7. Verify Submitted, Pending approval etc. by using keyboard.

**Actual Result:**
Filter controls like Submitted, Pending approval etc. are not accessible through keyboard under status lying on filters screen and due to this user is unable to interact with these controls and won’t be able to perform functionality over such controls.

**Expected Result:**
Filter controls like Submitted, Pending approval etc. should be accessible through keyboard under status lying on filters screen so that user could easily interact with these controls and could perform functionality over these controls.

**Note:** 
This issue also exists with chrome. 
[MAS2.1.1_Controls ar not accessible.zip](https://github.com/OfficeDev/office-ui-fabric-react/files/3599478/MAS2.1.1_Controls.ar.not.accessible.zip)
